src/mono/netcore/Makefile

```diff
@@ -14,28 +15,39 @@ ifeq ($(words $(wildcard ../../../artifacts/tests/coreclr/*.*.*)), 1)
 	CORECLR_TESTS_CONFIG := $(word 3,$(subst ., ,$(notdir $(wildcard ../../../artifacts/tests/coreclr/*.*.*))))
 endif
 
+ifeq ($(words $(wildcard ../../../artifacts/bin/testhost/*-*-*-*)), 1)
+	COREFX_TESTS_CONFIG := $(word 3,$(subst -, ,$(notdir $(wildcard ../../../artifacts/bin/testhost/*-*-*-*))))
+endif
+
 MONO_PROJ=/p:CoreClrTestConfig=$(CORECLR_TESTS_CONFIG) /p:Configuration=$(MONO_RUNTIME_CONFIG) ../mono.proj
 
 # run sample using local .dotnet-mono
 # build it with .dotnet first in order to be able to use LLVM only for the actual sample
 run-sample: patch-mono-dotnet
 	$(DOTNET) build -c Release sample/HelloWorld
-	MONO_ENV_OPTIONS="" COMPlus_DebugWriteToStdErr=1 \
+	MONO_ENV_OPTIONS="--aot" COMPlus_DebugWriteToStdErr=1 \
```

Comment:
why do we want to default to --aot here?

Reply:
It's supposed to be `--llvm` (just to showcase how to use mono cli). If mono runtime is built without llvm it won't crash, will just work normally + print "runtime doesn't support llvm"